lib/translator/common.h

```diff
@@ -19,15 +19,31 @@
 #include <type_traits>
 
 #include "absl/types/span.h"
+#include "lib/scsi_defs.h"
+#include "third_party/spdk_defs/nvme_defs.h"
```

Comment:
nit: now that clang-format has `IncludeBlocks: preserve` we should probably have separate groups for other project's .h files, including third_party, and our own project's .h files. 

Although the style guide doesn't indicate a blank line between those groups, it later also says to have a blank line between each non-empty group. And in an earlier PR, I received a comment to separate them.